### DISCUSSION TITLE
Severe bug / repro script -> does not visually show last object

### DISCUSSION THREAD

**Discussion by richardrl** [bulletphysics/bullet3#3933] [General]
[test.zip](https://github.com/bulletphysics/bullet3/files/6965197/test.zip)
[d435.zip](https://github.com/bulletphysics/bullet3/files/6965201/d435.zip)

If you try to run this, it won't show the last object in the visualizer, no matter how many objects you load.

**Comment by erwincoumans** [comment]
PyBullet has some limit in the amount of vertices, you may need to simplify the model, reducing the number of vertices and faces, for example using Blender (mesh decimation).
